ui/selectors/selectors.js

```diff
@@ -1635,6 +1635,10 @@ export function getRemoveNftMessage(state) {
   return state.appState.removeNftMessage;
 }
 
+export function getDappPermissionModal(state) {
```

Comment:
I don't think it should be a selector. We can either do it using useState in local state?